pcsx2/gui/Panels/PluginSelectorPanel.cpp

```diff
@@ -669,14 +670,11 @@ void Panels::PluginSelectorPanel::OnEnumComplete( wxCommandEvent& evt )
 
 	int emptyBoxes = 0;
 	const PluginInfo* pi = tbl_PluginInfo;
+	const PluginsEnum_t pid = pi->id;
 
-	do
+	if (pi->shortname != 0)
 	{
-		const PluginsEnum_t pid = pi->id;
-		if( m_ComponentBoxes->Get(pid).GetCount() <= 0 )
-			emptyBoxes++;
-
-		else if( m_ComponentBoxes->Get(pid).GetSelection() == wxNOT_FOUND )
+		for (auto& file : fs::directory_iterator(PathDefs::GetPlugins()))
```

Comment:
This looks wrong. The original code iterates through the combobox entries. This code seems to iterate through the files in the plugin directory instead.

Reply:
Yeah it does. I swapped it out for a file iterator to check if the plugins exist in the folder and list them.

Reply:
IIrc there was some issues with plugin enumeration going through WX so I had to scrap the whole thing